.github/workflows/publish.yml

```diff
@@ -0,0 +1,157 @@
+name: Publish
+
+env:
+  CLOUDSMITH_REPO: "ibm-dev"
+  RELEASE_VERSION: "1"
```

Comment:
Is this going to be manually incremented if we need to re-generate packages for a given FULL_VERSION?

Reply:
Yes we'd need to increment this manually if we wanted to publish new packages. It's a bit clumsy since we'd then need to remember to reset it for the next release were we ever to actually increment it though it does give us a canonical record of what the current package iteration is.

I was thinking about this and your suggestion to use `on: workflow_dispatch` - this would allow us to make the release version a parameter instead which would be set on a per-run basis (defaulting to `1`) which might be better. The downside is that we'd then need to remember the previous value were we to republish packages again, though this could be tracked in the news file (to be added in a subsequent PR).

Either way it's probably something worthy of a step in the release process.